projects/plugins/jetpack/modules/contact-form/grunion-contact-form.php

```diff
@@ -202,6 +204,17 @@ protected function __construct() {
 			)
 		);
 
+		add_menu_page(
+			__( 'Feedback', 'jetpack' ),
+			__( 'Feedback', 'jetpack' ),
+			'edit_pages',
+			$slug,
+			null,
+			'dashicons-feedback',
+			45
+		);
```

Comment:
I get the following error when testing this branch on my local Jetpack site:

```
Fatal error: Uncaught Error: Call to undefined function add_menu_page()
in /usr/local/src/jetpack-monorepo/projects/plugins/jetpack/modules/contact-form/grunion-contact-form.php on line 207
```

Seems the `add_menu_page` function is not available in the class `Grunion_Contact_Form_Plugin` constructor. I think this can be solved by moving this piece of code to `rename_feedback_menu`.

Reply:
Thanks @mmtr for testing it. I couldn't replicate that error but I've moved that code into the rename function.